daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -285,10 +351,31 @@ def yum_install(g, *packages):
       packages, last_exception))
 
 
+def customize_error_message(e: BaseException,
+                            errs: NonFatalErrors) -> BaseException:
+  """Create an error message based on PotentialErrors."""
+  logging.debug('Translation failed due to: {}'.format(e))
+  logging.debug('Potential errors: {}'.format(errs))
+  if errs.yum_command_not_found:
+    return RuntimeError('Verify the disk\'s OS: `yum` not found. {}'.format(e))
+  if errs.subscription_manager_status_error:
+    return RuntimeError(
+        'subscription-manager did not find an active subscription. Omit '
+        '`-byol` to register with on-demand licensing. {}'.format(e))
+  if errs.yum_unreachable_repo:
+    return RuntimeError('Ensure that all configured repos are reachable. '
+                        '{}'.format(e))
```

Comment:
Can multiple potential causes happen at the same time? If so, this logic will only return the first one.

Reply:
Theoretically multiple errors could occur. I'd like to try this simple version first, though, since it's a big step forward from our current error messages. 

Good news: Given this PR, it's a straightforward change to identify multiple errors in a future modification if we find that helpful.